tests/cli/test_cli_install.py

```diff
@@ -92,3 +94,24 @@ def test_find_conflicts_called_once(
             *channels,
         )
     assert mocked_find_conflicts.call_count == 3
+
+
+@pytest.mark.integration
+def test_emscripten_forge(
+    tmp_path: Path,
+    monkeypatch: MonkeyPatch,
+    tmp_env: TmpEnvFixture,
+):
+    monkeypatch.setenv("CONDA_PKGS_DIRS", str(tmp_path))
+    reset_context()
+
+    with tmp_env(
+        "--platform=emscripten-wasm32",
+        "--override-channels",
+        "-c",
+        "https://repo.mamba.pm/emscripten-forge",
+        "-c",
+        "conda-forge",
+        "pyjs",
```

Comment:
I see you removed the mocker patch but we still need to add `--yes`:

```suggestion
        "--channel=https://repo.mamba.pm/emscripten-forge",
        "--channel=conda-forge",
        "pyjs",
        "--yes",
```